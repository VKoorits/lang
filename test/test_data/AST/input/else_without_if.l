i = int(random(0,1))
if i:
	print("orel\n")
print("first player win")
else:
	print("reshka\n")
	print("second player win")

