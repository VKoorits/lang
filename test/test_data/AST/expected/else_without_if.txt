ERROR: expectedd 'if' before 'else' construction
